msteams-platform/bots/how-to/bot-configuration-experience.md

```diff
@@ -54,135 +65,196 @@ You must configure the `fetchTask` property under `bots.configuration` object in
-                      }
-                  }
-              }
-          }
-      }
-      else {
-    
-          try {
-              return {
-                  status: 200,
-                  body: {
-                      config: {
-                          type: 'message',
-                          value: "end"
-                      }
-                  }
-              }
-          }
-    ```
+When a user installs the bot in channel or group chat, the `fetchTask` property in the app manifest file initiates either `config/fetch` or `config/submit` as defined in the `teamsBot.js` file.
 
-## Bot configuration experience in Teams
+If you set the `fetchTask` property in the app manifest to:
 
-After you've created a bot to enable the bot configuration settings from a team or group chat scope, the user can configure and reconfigure the bot in Teams.
+* **false**: The bot doesn't fetch a dialog or an Adaptive Card. Instead, the bot must provide a static dialog or card that is used when the bot is invoked. For more information, see [dialogs](../../task-modules-and-cards/what-are-task-modules.md).
 
-To configure the bot, follow these steps:
+* **true**: The bot initiates either `config/fetch` or `config/submit` as defined. When the bot is invoked, you can return an Adaptive Card or a dialog depending on the context provided in [channelData and userdata](../../messaging-extensions/how-to/action-commands/create-task-module.md#payload-activity-properties-when-a-dialog-is-invoked-from-a-group-chat).
 
-1. Go to **Microsoft Teams**.
+The following table lists the response type associated with the invoke requests:
 
-1. Select **Apps**.
+|Invoke request |Response type |
+| --- | --- |
+| `config/fetch` | `Type: "continue"` or `Type = "auth"` |
+| `config/submit` | `Type: "continue"` or `Type: "message"` |
 
-1. From the Teams store, select a bot app you want to install.
+* `type: "continue"`: `type: "continue"` is used to define a continuation of a dialog or Adaptive Card within a bot configuration. When the type is set to `continue`, it indicates that the bot is expecting further interaction from the user to continue with the configuration process.
 
-1. From the dropdown next to **Add**, select **Add to a team** or **Add to a chat**.
+   The `adaptiveCardForContinue` is a custom function that returns the JSON for an Adaptive Card to be used in different stages of a bot’s workflow. These functions are used to return Adaptive Cards for different scenarios based on the user’s interaction with the bot.
 
-   :::image type="content" source="../../assets/images/bots/group-chat-add-Bot.png" alt-text="Screenshot shows add your bot to chat.":::
+   When the user submits the configuration, the `OnTeamsConfigSubmitAsync` method is triggered. It reads the user's input and returns a different Adaptive Card. You can also update the bot configuration to return a [dialog](../../task-modules-and-cards/what-are-task-modules.md).
```

Comment:
Let's not say "the `OnTeamsConfigSubmitAsync` method is triggered", let's say a config/submit invoke is triggered. Because OnTeamsConfigSubmitAsync is only the name of C#, in JS it's called handleTeamsConfigSubmit.